### PR TITLE
Add reclaim transactions to the stress test

### LINKED ISSUE #734: Add reclaim transactions to the stress test
### Description

Extend the stress tests to post reclaim transactions from the known legacy accounts.

### Acceptance Criteria
- Add the passphrases for the known legacy accounts in `config/{dev,alpha}net/known_legacy_accounts.json`
- Script is updated to post reclaim transactions
- Replace `isValidatorsRegistered, isSidechainRegistered` with `roundNumber` in `start` and adjust the logic accordingly
- The following transactions should be executed only for `roundNumber === 0`
  - sendRegisterKeysTransaction
  - sendSidechainRegistrationTransaction
  - sendReclaimTransaction

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #734 

### How was it solved?

- [x] Add the passphrases for the known legacy accounts in config/{dev,alpha}net/known_legacy_accounts.json
- [x] Script is updated to post reclaim transactions
- [x] Replace isValidatorsRegistered, isSidechainRegistered with roundNumber in start and adjust the logic accordingly
- [x] The following transactions should be executed only for roundNumber === 0
  - [x] sendRegisterKeysTransaction
  - [x] sendSidechainRegistrationTransaction
  - [x] sendReclaimLSKTransaction

### How was it tested?
- npm install --registry https://npm.lisk.com/
- npm run build
- ./bin/run start -n devnet --api-ipc
- npm run test:stress -n devnet

### CODE REVIEW

File: test/scripts/stress_test.ts [hunk 1]
```diff
@@ -111,6 +113,31 @@ const start = async (count, isValidatorsRegistered, isSidechainRegistered) => {
 		// Wait for 2 blocks
 		console.log('\n');
 		await wait(20000);
+
+		// require known legacy accounts based on the network, default to devnet
+		const { legacyAccounts } = require(`../../config/${network}/known_legacy_accounts.json`);
```
Comment (ManuGowda) [inline test/scripts/stress_test.ts:118]: we can move the `known_legacy_accounts.json` under `test/config`?

**Review by ManuGowda — COMMENTED** [review]
minor comment

File: test/scripts/stress_test.ts (hunk 1 above)
Comment (nagdahimanshu) [reply test/scripts/stress_test.ts]: Done!

**Review by sameersubudhi — APPROVED** [review]
Works as expected 👍

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**